components/script/dom/document.rs

```diff
@@ -2534,7 +2534,7 @@ impl Document {
         self.window
             .webgl_chan()
             .expect("Where's the WebGL channel?")
-            .send(WebGLMsg::SwapBuffers(dirty_context_ids, sender))
+            .send(WebGLMsg::SwapBuffers(dirty_context_ids, sender, 0))
```

Comment:
Let's send a real time here; having 0 kept giving me weird output I had to ignore.